### PR TITLE
add dropout to custom model in Part 2 tutorial

### PR DESCRIPTION
**Pull request by arshjot**
Solves #35 
Added dropout in custom model by using the optional ```training``` argument in ```call```, as shown [here](https://www.tensorflow.org/api_docs/python/tf/keras/Model)

### CODE REVIEW

File: examples/Part 02 - Intro to Private Training with Remote Execution.ipynb [hunk 1]
```diff
@@ -36,7 +36,9 @@
   {
```
Comment (jvmancuso) [inline examples/Part 02 - Intro to Private Training with Remote Execution.ipynb]: <p>I believe the <code>if training:</code> conditional is incorrect -- if the layer isn't included at inference time, they might not do the scaling by <code>1/p</code> that's needed to correct for the dropout. TF should take care of switching to inference mode just by passing that parameter to the dropout layer as you've already done</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft-TensorFlow/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (arshjot) [reply examples/Part 02 - Intro to Private Training with Remote Execution.ipynb]: I guess you are correct. I removed the conditional and checked that we get the same training/inference results from both custom and sequential models now.
